web-frontend/cypress/e2e/acceptance/home/send.cy.tsx

```diff
@@ -25,24 +31,44 @@ describe('E2E | Acceptance | Home', () => {
       mockedAccounts.forEach((account) => {
         server.createList('asset', 3, { account });
       });
+
       const account = {
-        nickname: 'Mario',
+        nickname: 'MarioX',
+        candidateBalance: '1000000000000',
+        balance: '10000',
         address: 'AUHdadXyJZUeINwiUVMtXZXJRTFXtYdihRWitUcAJSBwAHgcKAjtxx',
       };
       mockedAccounts.push(server.create('account', { ...account }));
-
       return mockedAccounts;
     }
 
     beforeEach(() => {
       mockedAccounts = mockAccounts();
     });
 
+    function navigateToHome() {
+      cy.visit('/');
+      cy.get('[data-testid="account-2"]').click();
+    }
+
+    function navigateToTransfercoins(index) {
+      cy.visit('/');
+
+      cy.get(`[data-testid="account-${index}"]`).click();
+      cy.get('[data-testid="side-menu"]').click();
+      cy.get('[data-testid="side-menu-sendreceive-icon"]').click();
+    }
+
+    function setAccountBalance(account) {
+      const balance = Number(account?.candidateBalance || 0);
```

Comment:
We are setting **candidateBalance** on line 37. It will always have. No need the OR condition.